nbdime/prettyprint.py

```diff
@@ -72,9 +72,9 @@ def present_dict_no_markup(prefix, d, exclude_keys=None):
                 pp.append(prefix + key + ':')
                 pp.extend(present_value(value_prefix, value))
             else:
-                pp.append(prefix + '%s: %s' % (key, value))
+                pp.extend(present_value(prefix + key + ': ', value))
         else:
-            pp.append(prefix + '%s: %s' % (key,  value))
+            pp.extend(present_value(prefix + key + ': ', value))
```

Comment:
Nevermind. I'll incorporate the features you add below in this file in the new prettyprinting code so when the merge comes I can just drop your version and pick mine. You don't have to change this PR.